content/tutorial/tutorial.md

```diff
@@ -59,12 +59,18 @@ You can now skip the second setup option, and go to the [Overview](#overview) se
 
 ### Setup Option 2: Local Development Environment
 
-**This is completely optional and not required for this tutorial!**
+This is completely optional and not required for this tutorial!
+
+<br>
+
+<details>
+
+<summary><b>Optional: Click to expand the local development instructions</b></summary>
```

Comment:
"local development instructions" is a bit vague. Something like this maybe:

"Optional: Instructions for following along locally using your preferred text editor"